conf/test.config

```diff
@@ -18,14 +19,11 @@ params {
   // Input data
   FW_primer = "GTGYCAGCMGCCGCGGTAA"
   RV_primer = "GGACTACNVGGGTWTCTAAT"
-  classifier = "https://github.com/nf-core/test-datasets/raw/ampliseq/testdata/GTGYCAGCMGCCGCGGTAA-GGACTACNVGGGTWTCTAAT-gg_13_8-85-qiime2_2019.7-classifier.qza"
-  metadata = "https://github.com/nf-core/test-datasets/raw/ampliseq/testdata/Metadata.tsv"
-  outdir = "./results"
-  temp_dir = "./results/tmp_dir"
-  readPaths = [
-    ['1_S103', ['https://github.com/nf-core/test-datasets/raw/ampliseq/testdata/1_S103_L001_R1_001.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/ampliseq/testdata/1_S103_L001_R2_001.fastq.gz']],
-    ['1a_S103', ['https://github.com/nf-core/test-datasets/raw/ampliseq/testdata/1a_S103_L001_R1_001.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/ampliseq/testdata/1a_S103_L001_R2_001.fastq.gz']],
-    ['2_S115', ['https://github.com/nf-core/test-datasets/raw/ampliseq/testdata/2_S115_L001_R1_001.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/ampliseq/testdata/2_S115_L001_R2_001.fastq.gz']],
-    ['2a_S115', ['https://github.com/nf-core/test-datasets/raw/ampliseq/testdata/2a_S115_L001_R1_001.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/ampliseq/testdata/2a_S115_L001_R2_001.fastq.gz']]
-  ]
+  outdir = "./results_test"
+  input = "https://github.com/nf-core/test-datasets/raw/ampliseq/samplesheets/Samplesheet.tsv"
```

Comment:
It was discussed that the CSV format most likely is preferred by nf-core in the future but the point for TSV is that its locale agnostic, i.e. a tab always means a tab, the emaning of a comma is locale dependent. This was acknowledged. Doesn't really matter, it would be easily possible to add a parser for CSV as well. Right now `--input` allows three kind of inputs (samplesheet, sequences, or folder) dependent on 4 different file endings.